canvas.h

```diff
@@ -59,31 +65,42 @@ class Canvas : public ICanvas
         return _effects;
     }
 
-    vector<unique_ptr<ILEDFeature>>& Features() override
+    vector<ILEDFeature *> Features() override
```

Comment:
If `unique_ptr` is no longer usable for this scenario, wouldn't it be better to use a `shared_ptr` instead?